### PR TITLE
Add issue & PR template in github

### PR DESCRIPTION
**Pull request by kerthcet**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#### Does this PR introduce a user-facing change?
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+-->
+```release-note
+
+```
```
Comment (sanposhiho) [inline .github/PULL_REQUEST_TEMPLATE.md:48]: Do we need this? I'm not sure if the bot supports the release note generation feature on sigs repo.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide/first-contribution.md#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
```
Comment (sanposhiho) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Also, let's have a link for https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/main/CONTRIBUTING.md.

It's now a default one tho, we can improve later.

**Review by sanposhiho — COMMENTED** [review]
Thanks

**Review by sanposhiho — COMMENTED** [review]
Thanks

**Review by sanposhiho — COMMENTED** [review]
Thanks

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (kerthcet) [reply .github/PULL_REQUEST_TEMPLATE.md:48]: This is synced with the k/k repo.

Comment (kerthcet) [reply .github/PULL_REQUEST_TEMPLATE.md:48]: At least better to know the impact of the PR? If you disagree, I can remove.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (kerthcet) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Copied from the k/k repo, just removed the last part about the docs.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (sanposhiho) [reply .github/PULL_REQUEST_TEMPLATE.md]: I checked the behavior of the bot in:
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/pull/4

It even doesn't add `release-note` related labels. 

But, I agree that we should have a section to explain what's the expected user facing changes.
So, can you simplify this section like:

```md
#### Does this PR introduce a user-facing change?
<!--
Enter a brief explanation of what are the expected user-facing changes by this PR.
-->

```

Comment (kerthcet) [reply .github/PULL_REQUEST_TEMPLATE.md]: We can generate release note following https://github.com/kubernetes/release/tree/master/cmd/release-notes
So let keep the same with k/k to work well with release-note cli.

Comment (sanposhiho) [reply .github/PULL_REQUEST_TEMPLATE.md]: Oh, sounds great. Yep, then let's go with it.

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

**Pull request merged**